<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAssignmentExceptionClass</class>
 <widget class="QDialog" name="DialogAssignmentExceptionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogAssignmentException</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Exceptions</string>
   </property>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
	background: transparent;
}

QHeaderView::section {	
	color: white; 
	background-color: rgba(70, 133, 210, 1);
}

QTableWidget::item:selected {
    background-color: #0e6c96;
	color: #fff;
}</string>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Paid Hours</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Work Coefficient</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>471</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Assignment</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="from">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="to">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Days of week</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>371</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="all_days">
       <property name="text">
        <string>All days of week</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="thursday">
       <property name="text">
        <string>Thursday</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="monday">
       <property name="text">
        <string>Monday</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="wednesday">
       <property name="text">
        <string>Wednesday</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="tuesday">
       <property name="text">
        <string>Tuesday</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="friday">
       <property name="text">
        <string>Friday</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="saturday">
       <property name="text">
        <string>Saturday</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBox_8">
       <property name="text">
        <string>Sunday</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To employee</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget { border: 1px solid #ccc; }</string>
    </property>
    <widget class="QRadioButton" name="all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="deparment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filter_deparment">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>39</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="department_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>69</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="department_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>349</width>
        <height>59</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="selection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Selection</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filter_employee">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="employee_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="employee_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>349</width>
        <height>109</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
    </property>
    <property name="text">
     <string>Apply Changes</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/check.png</normaloff>../icon/check.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
